### ISSUE TITLE
Migrate subscriptions to new site?

### ISSUE THREAD

**Issue by tevyaw** [stcr/subscribe-to-comments-reloaded#204]
Hi, we're rebuilding a site from scratch to get rid of years of cruft. But we want to export and import all our posts, comments, and keep our subscribe to comments reloaded subscriptions in-tact. If we do the default WP export/import process, will it keep all the StCR subscriptions in there and linked to their respective posts? Or is there some way to export and import it?

I appreciate any direction here. Thanks.

**Comment by Reedyseth** [comment]
> If we do the default WP export/import process, will it keep all the StCR subscriptions in there and linked to their respective posts?

@tevyaw My answer will be, it depends. Your comments are store in your `prefix_postmeta` table and the StCR plugin settings are store in the `prefix_options` table, so if the WP export/import process looks at all the information in those tables you are good to go.

Let me know if that works for you,

Regards.!!

**Comment by tevyaw** [comment]
Yes, it appears they all got migrated over in-tact. Unfortunately my images did not. :( 

Oh well, that's not your issue. Thanks! 

If anyone else is looking at this, it's likely it'll only migrate the StCR data, if you do a full export of everything.

**Comment by Reedyseth** [comment]
I guess many people will benefit for having a built in importer/exporter.
So I my add that feature probably soon than expected.

**Comment by vertopia** [comment]
> I guess many people will benefit for having a built in importer/exporter.
> So I my add that feature probably soon than expected.

I have the same need @Reedyseth This will be a great and welcome feature! Any idea when we can expect this?

Thanks!
